### ISSUE TITLE
Install Command

### ISSUE THREAD

**Issue by weaksauce** [Carthage/Carthage#990]
Hello, I would like to suggest having an install command added that only installs the newly added items to the cartfile but does not build/update the ones that are already there and built.

**Comment by Austinate** [comment]
Looks like https://github.com/Carthage/Carthage/pull/886 is exactly about this suggestion.

**Comment by ikesyo** [comment]
Once #972 is merged, you could do as follows:

```bash
carthage update NewDependency
```

**Comment by weaksauce** [comment]
@Austinate and @ikesyo, those are similar but not quite what I was asking for... I was asking for something akin to ruby bundler's ```bundle install``` command. That command will not touch any gems (libraries) that have been already resolved by the Gemfile.lock (Cartfile.lock) but will install any new gems that have been added to the Gemfile(Cartfile). In carthage it's a bit easier because there is no dependency management going on behind the scenes where they are searching for compatible gems that work with all the other gems. 

pseudocode:

```
for dependency in Cartfile.lines{
   if !Carthage.Installed?(dependency){

**Comment by ikesyo** [comment]
#972 does the thing, I think. Existing dependencies which are not in the arguments would not be updated nor built.

**Comment by weaksauce** [comment]
@ikesyo, it's close but not quite what I am asking for unless I am missing something. 

Cartfile:

```
github 'foo/foo' #installed already
github 'bar/bar'
github 'baz/baz'

```

Your pull request: ```carthage update bar baz```

My desire: ```carthage install```

Both of the above commands would fetch and build bar and baz and add them to the lock file while leaving foo unchanged and not built.

**Comment by mdiep** [comment]
This is a duplicate of #342. :smile:
